examples/00_quick_start/sar_movielens.ipynb

```diff
@@ -497,7 +497,7 @@
    "source": [
     "### 2.3. Evaluate how well SAR performs\n",
     "\n",
-    "We evaluate how well SAR performs for a few common ranking metrics provided in the `python_evaluation` module in reco_utils. We will consider the Mean Average Precision (MAP), Normalized Discounted Cumalative Gain (NDCG), Precision, and Recall for the top-k items per user we computed with SAR. User, item and rating column names are specified in each evaluation method."
```

Comment:
Revert this.

Reply:
wow, this is so weird, don't understand why those lines were deleted

Reply:
They were not deleted, you just changed a single word.